Image_process/processImage.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
+        self.image_path = image_path
+
+    def resize_image(self):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        width = 128
+        height = 128
+        #if image isn't square raise a warning and resize to square
+        image = Image.open(self.image_path)
+        if image.size[0] != image.size[1]:
+            print("Warning: Image is not square. Resizing to square.")
+
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
```

Comment:
assuming `save_path` to be a string will lead to issues when implementing saving files to s3. the second parameter of this function should be a path-like (`str`, `pathlib.Path`) or file-like (`io.BytesIO`) object